index.js

```diff
@@ -0,0 +1,3 @@
+function sum(a, b) {
```

Comment:
Arguments should not be named a and b. Try using names that associate with the meaning of the function.